tests/unit/test_config.py

```diff
@@ -126,8 +126,10 @@ def test_env_config_missing_file_raises(monkeypatch):
         config._get_config_files()
 
 
-def test_env_config_not_defined_does_not_raise(mock_clean_env):
-    assert config._get_config_files() == []
+def test_env_config_not_defined_does_not_raise(mock_clean_env, monkeypatch):
+    with monkeypatch.context() as m:
+        m.setattr(config, "_DEFAULT_FILES", [])
+        assert config._get_config_files() == []
```

Comment:
Since we're only setting the attribute and not mocking any library, I think the context manager here might not be needed, WDYT?

I'm happy to merge this as is though.

Reply:
I like it as it is. My worry is that if we set the attribute it will also affect other tests which use that attribute. I guess I would rather be safe...